scala-exercises-client/src/main/scala/config/Routes.scala

```diff
@@ -2,13 +2,9 @@ package config
 
 object Routes {
 
-  object Users {
-    val base = "/users"
-    def all = base + "/all"
-    def create = base + "/create"
-    def update(id: Long) = base + s"/update/$id"
-    def delete(id: Long) = base + s"/delete/$id"
-    def clear = base + "/clear"
+  object Exercises {
+    val base = "/exercises/sections"
+    def evaluate(section: String, category: String) = base + s"/$section/$category/evaluate"
```

Comment:
Can't we use the Play's reverse routes here instead of recreating a Routes file manually?